src/utils/ReduxUtils.ts

```diff
@@ -38,9 +44,12 @@ export interface IReduxAction<T> extends Redux.Action {
 
 export interface IReduxActionPayload {
   id?: string;
-  filterText?: string;
+  ids?: string[];
   facet?: string;
   facetRow?: IFacet;
+  filterText?: string;
+  pageNb?: number;
```

Comment:
Actually, reducers listen to all actions. I could for example listen to a loading event in my pagination reducer. Same goes for dispatch actions in mapDispatchToProps. Each action still does have its own payload.